spec/models/generic_file_spec.rb

```diff
@@ -625,4 +625,156 @@
       end
     end
   end
+
+  let(:public_permission_double) { double("Hydra::AccessControls::Permission") }
+  let(:gv_black_photo_sub_collection_double) { double("Collection") }
+  let(:gv_black_collection_double) { double("Collection") }
+  let(:random_doi) { "10.18131/g3-nqgv-4863" }
+  let(:blank_doi) { "" }
+  let(:public_record) { GenericFile.new(doi: []) }
+  let(:private_record) { GenericFile.new(doi: []) }
+
+  # if record is publc, has a doi, and is either in the gv black photograph sub collection or is not in the greater gv black collection
+  describe "#unexportable?" do
+    before do
+      allow(public_permission_double).to receive(:agent).and_return(GenericFile::PUBLIC_PERMISSION)
+
+      allow(gv_black_photo_sub_collection_double).to receive(:map).and_return([GenericFile::GV_BLACK_PHOTOGRAPH_SUB_COLLECTION_ID])
+      allow(gv_black_photo_sub_collection_double).to receive(:id)
+
+      allow(gv_black_collection_double).to receive(:map).and_return([GenericFile::GV_BLACK_COLLECTION_ID])
```

Comment:
```suggestion
      allow(gv_black_collection_double).to receive(:map).and_return([GenericFile::GV_BLACK_PHOTOGRAPH_SUB_COLLECTION_ID])
```

no ?

Reply:
I think Im misunderstanding something here. I thought it made sense for `gv_black_photo_sub_collection_double` to use the ID that ties to that same collection and similarly that the `gv_black_collection_double` should return the ID that collection. Is there any reason you want to switch these around?